addons/test_mass_mailing/tests/test_blacklist.py

```diff
@@ -8,15 +8,83 @@
 
 class TestBLAccessRights(common.MassMailingCase):
 
+    def setUp(self):
+        super(TestBLAccessRights, self).setUp()
+        self.bl_rec = self.env['mail.blacklist'].create([
+            {'email': 'Not A Stark <john.snow@example.com>'},
+        ])
+
+        self.user_portal = self.env['res.users'].with_context(self._quick_create_user_ctx).create({
+            'name': 'Patricia Portal',
+            'login': 'port',
+            'email': 'p.p@example.com',
+            'signature': '--\nPatricka',
+            'notification_type': 'email',
+            'groups_id': [
+                (6, 0, [self.env.ref('base.group_portal').id]),
+            ],
+        })
+
+        self.bl_previous = self.env['mail.blacklist'].search([])
+
     @users('emp')
     def test_bl_create_employee(self):
         with self.assertRaises(AccessError):
             self.env['mail.blacklist'].create([{'email': 'Arya.Stark@example.com'}])
 
+    @users('emp')
+    def test_bl_cead_employee(self):
```

Comment:
Typo: s/cead/read (also repeated below).

Also, is there any reason no to group all tests executed by the same user under a single test method for all CRUD? Should mean less lines of code, and little difference in test result readability, no? Am I missing something? Even if we were able to execute each test method individually, there is little need to split the four CRUD tests I imagine.